### ISSUE TITLE
Can't find `felipe--kotlin-new-mpp-SNAPSHOT`

### ISSUE THREAD

**Issue by feresr** [felipecsl/ktnes#20]
I'm trying to get this project set up and to play around with it and I keep getting:

```
Could not find com.github.felipecsl.kotlin-frontend-plugin:kotlin-frontend:felipe--kotlin-new-mpp-SNAPSHOT.
```

```
Searched in the following locations:
  - file:/Users/feresr/.m2/repository/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/maven-metadata.xml
  - file:/Users/feresr/.m2/repository/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.module
  - file:/Users/feresr/.m2/repository/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.pom
  - https://jcenter.bintray.com/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/maven-metadata.xml
  - https://jcenter.bintray.com/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.module
  - https://jcenter.bintray.com/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.pom
  - https://dl.google.com/dl/android/maven2/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/maven-metadata.xml
  - https://dl.google.com/dl/android/maven2/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.module
  - https://dl.google.com/dl/android/maven2/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.pom
  - https://kotlin.bintray.com/kotlinx/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/maven-metadata.xml
  - https://kotlin.bintray.com/kotlinx/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.module
  - https://kotlin.bintray.com/kotlinx/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.pom
  - https://plugins.gradle.org/m2/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/maven-metadata.xml
  - https://plugins.gradle.org/m2/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.module
  - https://plugins.gradle.org/m2/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.pom
  - https://oss.sonatype.org/content/repositories/snapshots/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/maven-metadata.xml
  - https://oss.sonatype.org/content/repositories/snapshots/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.module
  - https://oss.sonatype.org/content/repositories/snapshots/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.pom
  - https://dl.bintray.com/kotlin/kotlin-eap/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/maven-metadata.xml
  - https://dl.bintray.com/kotlin/kotlin-eap/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.module
  - https://dl.bintray.com/kotlin/kotlin-eap/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.pom
  - https://dl.bintray.com/kotlin/kotlin-dev/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/maven-metadata.xml
  - https://dl.bintray.com/kotlin/kotlin-dev/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.module
  - https://dl.bintray.com/kotlin/kotlin-dev/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-SNAPSHOT.pom
  - https://jitpack.io/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/maven-metadata.xml
  - https://jitpack.io/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-0.0.37-g96c7e0f-3.module
  - https://jitpack.io/com/github/felipecsl/kotlin-frontend-plugin/kotlin-frontend/felipe--kotlin-new-mpp-SNAPSHOT/kotlin-frontend-felipe--kotlin-new-mpp-0.0.37-g96c7e0f-3.pom
Required by:
    project : > com.github.felipecsl:kotlin-frontend-plugin:felipe--kotlin-new-mpp-SNAPSHOT:0.0.37-g96c7e0f-3
```

**Comment by felipecsl** [comment]
Fixed by 226fcb9987f929f0e4313d5a9790be16ca557da0. Thanks for reporting!

**Comment by feresr** [comment]
Thank you for the super quick response!
